---
title: "The genomic landscape of Acute Respiratory Distress Syndrome: a meta-analysis by information content of genome-wide studies of the host response."
format: 
  pdf:
    include-in-header: 
      text: |
        \usepackage{lscape}
        \newcommand{\blandscape}{\begin{landscape}}
        \newcommand{\elandscape}{\end{landscape}}
    keep-tex: true
mainfont: Times New Roman
fontsize: "11"
mainfontoptions: 
  - Numbers=Lowercase
  - Numbers=Proportional
linestretch: "1.5"
cap-location: bottom
tbl-cap-location: top
fig-align: center
documentclass: article
papersize: "A4"
geometry: 
  - top=15mm
  - left=22.5mm
  - right=22.5mm
  - bottom=15mm
echo: false
bibliography: bib/ref_manuscript.bib
csl: bib/nature-medicine.csl
biblio-title: "References"
---

Jonathan E Millar^1^, Sara Clohisey-Hendry^1^, Megan McMannus^1^, Marie Zechner^1^, Bo Wang^1^, Nick Parkinson^1^, Melissa Jungnickel^1^, Nureen Mohamad Zaki^1^, Erola Pairo-Castineira^1^, Konrad Rawlik^1^, Clark D Russell^2^, Manu Shankar-Hari^2^, Carolyn Calfee^3^, Daniel F McAuley^4^, and J Kenneth Baillie^1^

1.  Roslin Institute, University of Edinburgh, Edinburgh, United Kingdom.
2.  Centre for Inflammation Research, University of Edinburgh, Edinburgh, United Kingdom.
3.  Division of Pulmonary, Critical Care, Allergy & Sleep Medicine, Department of Medicine, University of California San Francisco, San Francisco, USA.
4.  Wellcome-Wolfson Institute for Experimental Medicine, Queen's University Belfast, Belfast, United Kingdom.

<!-- To Do

- Functional enrichment text
- In-silico perturbation - analysis, text, and figure
- Sub-groups - analysis, text, and figure
- Methods - update tissue expression, functional enrichment, and in-silico perturbation
- Supplementary table for study details
- Supplementary data files

- Update website
- Update ARDSMAICR

-->

\newpage

## Abstract


\newpage

## Introduction

The acute respiratory distress syndrome (ARDS) is clinically defined as acute hypoxaemic respiratory failure due to non-cardiogenic pulmonary oedema[@ARDS_Definition_Task_Force]. It occurs following a variety of insults; pulmonary and extra-pulmonary. While this definition has been useful in identifying patients at risk of serious morbidity and death[@Bellani2016], it overlooks the underlying biology and masks heterogeneity[@Wilson2020]. Arguably, this has contributed to limited success in developing therapeutics[@Laffey2018]. In contrast, a biological definition of ARDS, based on mechanistically distinct sub-phenotypes, may provide the lever necessary for future drug discovery[@Bos2022].

Functional genomics technologies enable disease characterisation at unprecedented resolution. The emergence of coronavirus disease 2019 (COVID-19) has provided an opportunity to test their usefulness for drug discovery. A notable success has been the finding that baricitinib, a Janus kinase inhibitor, reduces mortality in patients hospitalised with COVID-19[@RECOVERYBari2022]. *A priori* support for baricitinib was greatly enhanced following the discovery of a causal link between elevated tyrosine kinase 2 (*TYK2*) expression and severe COVID-19 in genome-wide association studies (GWAS)[@Pairo-Castineira2021; @Kousathanas2022]. The availability of omics data for non-COVID ARDS is limited by comparison, although recent studies have used these techniques to examine signatures of non-COVID ARDS sub-phenotypes[@Bos2019; @Sarma2022].

An unresolved challenge is how large omics data can be effectively exploited[@Gomez-Cabrero2014]. Specifically, how can we combine data from heterogeneous sources to derive new insights or recalibrate our current understanding in the light of new data? We have proposed meta-analysis by information content (MAIC) as a data-driven, algorithmic, method for combining gene lists from diverse sources[@Li2020]. MAIC is agnostic to the quality or methodology of the sources and combines ranked or un-ranked gene sets by calculating weights for each list and gene, and iteratively updating them to converge on a ranked meta-list. We have successfully applied MAIC to host-genomics studies of Influenza A[@Li2020] and SARS-CoV-2[@Pairo-Castineira2021; @Parkinson2020], and shown that it out-performs existing algorithms when combining ranked and un-ranked lists obtained from heterogeneous sources[@Wang2022-jb].

Here we present a living meta-analysis by information content of ARDS host genomics studies as an open-source resource for gene prioritisation, translational genomics, and drug target discovery. A comprehensive, interactive interface is available at <https://baillielab.net/maic/ards>.

\newpage

## Results

#### Systematic review

Our search yielded 8,937 unique citations (Fig. S1). Of these, we retrieved 74 studies for full-text evaluation and ultimately included 40 in our meta-analysis[@Batra2022; @Bhargava2014; @Bhargava2017; @Bime2018; @Bos2019; @Bowler2004; @Chang2008; @Chen2013; @Chen2016; @Christie2012; @Dolinay2012; @Dong2013; @Englert2019; @Frenzel2011; @GuillenGuio2020; @Howrylak2009; @Jiang2020; @Juss2016; @Kangelaris2015; @Kovach2015; @Liao2021; @Lu2017; @Martucci2020; @Meyer2011; @Meyer2013; @Mirchandani2022; @Morrell2018; @Morrell2019; @Nick2016; @Nguyen2013; @Ren2016; @Sarma2022; @Scheller2019; @Shortt2014; @Wang2008; @Tejera2012; @Xu2021; @Zhang2021; @Zhang2022; @Zhu2017]. These studies produced 44 unique gene lists (22 transcriptomic, 13 proteomic, and 9 based on genome-wide association studies (GWAS); see @tbl-tab1). Three studies reported results from multiple methodologies[@Liao2021; @Mirchandani2022; @Sarma2022], and several used more than one tissue type[@Bowler2004; @Chen2016; @Kovach2015]. Excluding GWAS, 14 lists (40%) were from lung or airway samples, and 21 (60%) from blood. We could not retrieve one gene list[@Frenzel2011], and no whole-genome sequencing GWAS were found. Only 36% (n=8) of transcriptomic lists used next-generation sequencing. The earliest study was published in 2004[@Bowler2004], while almost half (n=19, 47.5%) were published in the last 5 years.

Most studies aimed to identify genes/proteins associated with ARDS susceptibility (n=27, 67.5%). The remainder examined associations with survival (n=6, 15%), sub-phenotype (n=4, 10%), disease progression (n=2, 5%), or severity (n=1, 2.5%). In total, studies included 6,856 ARDS patients. Supplementary Table 1 provides detailed study designs, demographics, and ARDS aetiology.

#### Meta-analysis by information content (MAIC)

First, we analysed the 43 available gene lists using MAIC. Lists were categorized by method (GWAS, transcriptomics, proteomics) and technique (e.g., RNA-seq, mass spectrometry; see Table 1). In total, we ranked 7,085 unique genes or SNPs, with a median of 27 genes per list (range 1-4,954). The top 100 ranked genes are summarized in @fig-fig1. Most genes were found in a single category (n=5,866, 82.8%); only 157 (2.2%) were identified in ≥ 3 categories, with a maximum of 5 categories (@fig-fig1). Similarly, few genes (n=362, 5.1%) were identified by > 1 method, with only *AKR1B10*, *HINT1*, *HSPG2*, *S100A11*, and *SLC18A1* present in transcriptomic, proteomic, and GWAS based lists. To prioritise genes, we used the unit invariant knee method[@Christopoulos2016-jb] to identify the inflection point in the MAIC score curve. This prioritised 1,306 genes with scores above this point (@fig-fig1). These genes were more likely to be found in ≥ 2 lists or categories and by > 1 method (@fig-fig1).

To assess the influence of individual lists, we calculated the information content (IC), reflecting the sum of gene scores across all lists (@fig-fig2), and the information contribution (ICtb), measuring the sum of gene scores contributing to a gene's overall MAIC score. To obtain relative values, we divided the IC/ICtb for each list by the total. This showed that only 10 lists (from 9 studies) contributed >1% of total information by either metric (Tab. S2). Notably, the RNA-seq list from Sarma et al.[@Sarma2022] accounts for >50% of the total IC and ICtb, a function of its length. To account for this, we normalised relative IC/ICtb by the number of genes per list. Along with the proportion of replicated genes, this provides an alternative perspective, with several proteomic lists ranking highly (@fig-fig2).

#### Comparison with existing ARDS sources and COVID-19

To contextualise the results of our meta-analysis, we evaluated the degree of overlap between the genes prioritised by MAIC and those from two established resources: BioLitMine[@Hu2020-dl], using an ARDS MeSH search, and the ARDS Database of Genes[@Quintanilla2021] (Fig. S2). BioLitMine identified 271 ARDS-associated genes, of which 142 (52.4%) were in our analysis. Almost half of the overlapping genes (n = 63, 44.4%) were ranked within our prioritizsd set (Tab. S3). Of the 239 genes catalogued in the ARDS Database, 177 (74.1%) were also present in our study. However, both sources contain some unsupported gene associations. 

\newpage

\blandscape

| **Year** | **Study**                      | **Focus**      | **Definition** | **N**^a^ | **Method**      | **Technique**  | **Tissue** | **Cell type** |
|----------|--------------------------------|----------------|----------------|----------|-----------------|----------------|------------|---------------|
| 2022     | Batra[@Batra2022]              | Survival       | Berlin         | 24       | Proteomics      | Other          | Blood      |               |
|          | Mirchandani[@Mirchandani2022]  | Susceptibility | Berlin         | 22       | Proteomics      | Mass Spec      | Blood      | Monocytes     |
|          |                                |                |                |          | Transcriptomics | Microarray     | Blood      | Monocytes     |
|          | Sarma[@Sarma2022]              | Sub-phenotype  | Berlin         | 41       | Proteomics      | Other          | TA         |               |
|          |                                |                |                |          | Transcriptomics | RNA-seq        | TA         |               |
|          |                                |                |                |          | Transcriptomics | scRNA-Seq      | TA         | Immune cells  |
|          | Zhang[@Zhang2022]              | Susceptibility | AECC           | 11       | Transcriptomics | RNA-Seq        | Blood      | Exosomes      |
| 2021     | Liao[@Liao2021]                | Survival       | Either         | 390      | GWAS            | Genotyping     | Blood      |               |
|          |                                |                |                |          | Transcriptomics | RNA-seq        | Blood      | PBMCs         |
|          | Martucci[@Martucci2020]        | Sub-phenotype  | None           | 11       | Transcriptomics | Microarray     | Blood      |               |
|          | Xu[@Xu2021]                    | Survival       | Berlin         | 105      | GWAS            | WES            | Blood      |               |
|          | Zhang[@Zhang2021]              | Susceptibility | Berlin         | 5        | Transcriptomics | RNA-seq        | Blood      |               |
| 2020     | Guillen-Guio[@GuillenGuio2020] | Susceptibility | Berlin         | 633      | GWAS            | Genotyping     | Blood      |               |
|          | Jiang[@Jiang2020]              | Susceptibility | Berlin         | 3        | Transcriptomics | scRNA-seq      | Blood      | PBMCs         |
| 2019     | Bos[@Bos2019]                  | Sub-phenotype  | Berlin         | 210      | Transcriptomics | Microarray     | Blood      |               |
|          | Englert[@Englert2019]          | Susceptibility | Either         | 11       | Transcriptomics | RNA-seq        | Blood      |               |
|          | Morrell[@Morrell2019]          | Survival       | AECC           | 36       | Transcriptomics | Microarray     | BALF       | AMs           |
|          | Scheller[@Scheller2019]        | Susceptibility | None           | 6        | Transcriptomics | RNA-seq        | BALF       | EVs           |
| 2018     | Bime[@Bime2018]                | Susceptibility | Either         | 232      | GWAS            | Genotyping     | Blood      |               |
|          | Morrell[@Morrell2018]          | Susceptibility | Berlin         | 35       | Transcriptomics | Microarray     | BALF       |               |
| 2017     | Bhargava[@Bhargava2017]        | Survival       | AECC           | 36       | Proteomics      | Mass Spec      | BALF       |               |
|          | Lu[@Lu2017]                    | Susceptibility | AECC           | 12       | Transcriptomics | Microarray     | Blood      |               |
|          | Zhu[@Zhu2017]                  | Susceptibility | Berlin         | 199      | Transcriptomics | Microarray     | Blood      |               |
| 2016     | Chen[@Chen2016]                | Severity       | AECC           | 7        | Proteomics      | Mass Spec      | BALF/Blood |               |
|          | Juss[@Juss2016]                | Susceptibility | Berlin         | 23       | Transcriptomics | Microarray     | Blood      | Neutrophils   |
|          | Nick[@Nick2016]                | Sub-phenotype  | AECC           | 121      | Transcriptomics | Microarray     | Blood      | Neutrophils   |
|          | Ren[@Ren2016]                  | Susceptibility | Berlin         | 14       | Proteomics      | Other          | Blood      |               |
| 2015     | Kangelaris[@Kangelaris2015]    | Susceptibility | Berlin         | 29       | Transcriptomics | Microarray     | Blood      |               |
|          | Kovach[@Kovach2015]            | Susceptibility | AECC           | 18       | Transcriptomics | Microarray     | BALF/Blood | AMs           |
| 2014     | Bhargava[@Bhargava2014]        | Progression    | AECC           | 22       | Proteomics      | Mass Spec      | BALF       |               |
|          | Shortt[@Shortt2014]            | Susceptibility | AECC           | 213      | GWAS            | WES            | Blood      |               |
| 2013     | Chen[@Chen2013]                | Susceptibility | Berlin         | 11       | Proteomics      | Mass Spec      | Blood      |               |
|          | Dong[@Dong2013]                | Progression    | None           | 14       | Proteomics      | Mass Spec      | BALF       | AMs           |
|          | Meyer[@Meyer2013]              | Susceptibility | Berlin         | 661      | GWAS            | Genotyping     | Blood      |               |
|          | Nguyen[@Nguyen2013]            | Susceptibility | AECC           | 30       | Proteomics      | Mass Spec      | BALF       |               |
| 2012     | Christie[@Christie2012]        | Susceptibility | AECC           | 812      | GWAS            | Genotyping     | Blood      |               |
|          | Dolinay[@Dolinay2012]          | Susceptibility | AECC           | 35       | Transcriptomics | Microarray     | Blood      |               |
|          | Tejera[@Tejera2012]            | Susceptibility | AECC           | 1400     | GWAS            | Genotyping     | Blood      |               |
| 2011     | Frenzel[@Frenzel2011]          | Survival       | AECC           | 46       | Proteomics      | Mass Spec      | BALF       |               |
|          | Meyer[@Meyer2011]              | Susceptibility | AECC           | 1241     | GWAS            | Genotyping     | Blood      |               |
| 2009     | Howrylak[@Howrylak2009]        | Susceptibility | AECC           | 13       | Transcriptomics | Microarray     | Blood      |               |
| 2008     | Chang[@Chang2008]              | Susceptibility | None           | 20       | Proteomics      | Mass Spec      | BALF       |               |
|          | Wang[@Wang2008]                | Susceptibility | AECC           | 8        | Transcriptomics | Microarray     | Blood      |               |
| 2004     | Bowler[@Bowler2004]            | Susceptibility | AECC           | 16       | Proteomics      | Mass Spec      | BALF/Blood |               |
: Summary of studies and gene lists included in the systematic review {#tbl-tab1}

```{=tex}
\begin{scriptsize}
a - The number of patients with ARDS included in each study.
Abbreviations: AECC - American-European Consensus Conference; AMs - Alveolar macrophages; BALF - Bronchoalveolar lavage fluid; EVs - Extracellular vesicles; GWAS - Genome-wide association study; MS - Mass spectometry; PBMCs - Peripheral blood mononuclear cells; TA - Tracheal aspirate; WES - Whole-exome sequencing. 
\end{scriptsize}
```

\elandscape

\newpage

![**Meta-analysis by information content**. (a) Heatmap of top 100 ranked genes showing MAIC score, highest score per category, and number of supporting lists. (b) UpSet plot of MAIC genes showing total numbers for each category combination, MAIC score distribution, and supporting lists. (b) Gene prioritization using the Unit Invariant Knee method. Intersection of lines identifies elbow point of best-fit curve. 1,306 genes in upper left quadrant were prioritied. (c) Strip plots comparing number of lists and categories/methods per gene between prioritized and deprioritized sets. ](./img/Figure_1.png){#fig-fig1}

For the BioLitMine search, we identified 4 such genes not initially found in the ARDS MAIC set after correcting historical gene symbol aliases. A further 104 were supported by a single publication. For the remaining 21, we obtained their 100 most co-expressed genes using ARCHS4[@Lachmann2018] (returning data for 18) and assessed the overlap with ARDS MAIC (Fig. S2). Two-thirds exhibited <50% overlap. Finally, we compared the overlap between the genes ranked by MAIC for ARDS and by a previous MAIC of the host response to COVID-19[@Parkinson2020] (Fig. S2). In total, 2,606 ARDS genes (36.8%) were also found in COVID-19, of which 143 were prioritized by both analyses (Fig. S2).

![**Attributing information in MAIC**. (a) Shared information content (IC) between gene lists. Links indicate absolute IC (sum of common gene scores) between studies. (b) Proportion of replicated genes. Circle diameter is logarithm (base 2) of gene number per list. (c) IC normalized by number of genes. Overlapping circles denote equal normalized IC and contribution (ICtb - sum of common gene scores contributing to MAIC), indicating all gene scores contributed to MAIC. (d) Shared IC between categories, scaled so links show fraction of total IC. (e) Shared IC between methods, scaled. (f) Shared IC between tissue types, scaled.](./img/Figure_2.png){#fig-fig2}

#### Tissue and cell-specific expression

Despite the dominance of blood sampling, the majority of genes included in the meta-analysis were identified in airways samples (n=5,847, 82.5%) (Fig. S3). This was true for the prioritised set of genes, however, here most were also identified in blood (n=818, 62.6%) (Fig. S3). For the genes solely found in blood sampling lists (n=1,238), almost three-quarters are known to be expressed in the lung using scRNA-seq data (≥ 5 normalised transcripts per million (nTPM)), with a quarter highly-expressed (≥ 100 nTPM) (Fig S2).



#### Functional enrichment 

#### In-silico perturbation

#### Sub-groups

![**Functional enrichment of prioritised genes**. (a) Significantly enriched Reactome terms (*P* < 0.01). Terms colored by parent class and size proportional to recall. (b) Euler diagram of the overlap of hub genes identified by five methods. MNC - Maximum Neighbourhood Component, MCC - Maximal Clique Centrality, DMNC - Density of MNC, EPC - Edge Percolated Component. (c) Protein-protein interaction (PPI) network of hub genes, clustered using the Markov Chain Algorithm. (d). Heatmap of common hub genes displaying tissue type(s), MAIC score, highest category score, supporting lists, and presence in the druggable genome.](./img/Figure_3.png){#fig-fig3}

\newpage

## Discussion

## Methods

The systematic review and meta-analysis protocol was registered with the International Prospective Register of Systematic Reviews (PROSPERO; CRD42022306270). The review is reported in compliance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines[@Page2021].

#### Search strategy and selection criteria

A detailed description of our search strategy and eligibility criteria is provided in the Supplementary Methods. Briefly, we searched MEDLINE, Embase, bioRxiv, medRxiv, the ARDS Database of Genes[@Quintanilla2021], and the NCBI Gene Expression Omnibus from inception to December 1^st^, 2022 without language restrictions. We also performed single-level backwards and forwards citation searches using SpiderCite[@Clark2020] and hand-searched recent review articles[@Battaglini2022; @Hernandez-Beeftink2019; @Reilly2017; @Zheng2022].

We included human genome-wide studies reporting associations between genes, transcripts, or proteins and ARDS susceptibility, severity, survival, or phenotype, accepting any contemporaneous ARDS definition. We excluded paediatric studies (age < 18 years), animal studies, *in-vitro* human ARDS models, candidate *in-vivo* or *in-vitro studies* (< 50 genes/proteins), candidate gene associations, and studies with < 5 patients per arm (except scRNA-seq).

#### Outcomes

We retrieved ranked lists of genes associated with the ARDS host response, preferring measures of significance and adjusted *P* values over raw *P* values when multiple ranking measures were used. We obtained both summary lists (all implicated genes) and author-defined subgroup lists. To combine subgroup lists into summary lists, we took the minimum *P* value or maximum effect size. We excluded genes below the author-defined threshold for significance/effect magnitude. If unavailable, we excluded genes with *P* > 0.05, z-score < 1.96, or log fold change < 1.5.

#### Study selection and data extraction

Article titles and abstracts from our search were stored in Zotero v6.0-beta (Corporation for Digital Scholarship, United States). Titles were initially screened by one author using Screenatron[@Clark2020]. Two authors then independently screened abstracts against eligibility criteria, with a third resolving inconsistencies. Full texts and supplements of eligible studies were retrieved and inclusion adjudicated by consensus.

Data were extracted by one author and cross-checked by a second. Gene, transcript, or protein identifiers were mapped to HGNC symbols or Ensembl/RefSeq equivalents if no HGNC symbol was available. Unannotated SNPs were searched in NCBI dbSNP. miRBase (University of Manchester, United Kingdom) provided miRNA symbols. For microarray probes without symbols, we used the DAVID Gene Accession Conversion tool (Laboratory of Human Retrovirology and Immunoinformatics, Frederick National Laboratory for Cancer Research, United States) to map them to HGNC symbols. We extracted information relating to study design, methodology, tissue/cell type, demographics, ARDS aetiology, risk factors, severity, and outcomes.

#### Meta-analysis by information content (MAIC)

The MAIC algorithm has been described in detail[@Li2020; @Parkinson2020; @Pairo-Castineira2021; @Wang2022-jb]. Full documentation and the source code are available at https://github.com/baillielab/maic. Briefly, MAIC combines ranked and unranked lists of related named entities, such as genes, from heterogeneous experimental categories, without prior regard to the quality of each source. The algorithm makes four key assumptions; (1) genes associated with ARDS exist as true positives, (2) a gene is more likely to be a true positive if it is found in more than one source, (3) the probability of being a true positive is enhanced if the gene appears in a list that contains a higher proportion of replicated genes, and (4) the probability is further enhanced if it is found in more than one category of experiment. Based on these assumptions, MAIC compares lists with each other, forming a weighting for each source based on its information content, which is then used to calculate a score for each gene. The output is a ranked list summarizing the total information supporting each gene's association with ARDS. We have shown MAIC outperforms available algorithms, especially with ranked and unranked heterogeneous data[@Wang2022-jb].

As our primary analysis, we performed MAIC on all summary gene lists, regardless of study focus. Lists were assigned categories based on their methodology and experimental technique: genome-wide association study (GWAS) - genotyping, GWAS - whole exome sequencing, transcriptomics - microarray, transcriptomics - RNA-sequencing (RNA-seq), transcriptomics - single cell RNA-seq (scRNA-seq), proteomics - mass spectometry, and proteomics - other. For secondary analyses, we performed MAIC on subsets of lists based on study focus (i.e., susceptibility to ARDS or survival/severity). 

For each MAIC iteration, we prioritised genes with sufficient evidentiary support for further study (i.e., the gene set before which information content diminished such that there was little/no corroboration for the remainder's ARDS association). We used the unit invariant knee method [@inflection; @Christopoulos2016-jb] to identify the elbow point in the best-fit curve of MAIC scores. Genes with values above this point were prioritized for downstream analyses.

#### ARDS literature and SARS-CoV-2 associations

We used BioLitMine[@Hu2020-dl] to query the NCBI Gene database for genes associated with the Medical Subject Heading (MeSH) term "Respiratory Distress Syndrome, Acute", generating a list of genes and publications. We descriptively compared the overlap between this list and the MAIC-ranked gene list. Similar comparisons were made between the ARDS MAIC results and the gene set in the ARDS Database of Genes[@Quintanilla2021] and a prior MAIC of SARS-CoV-2 host genomics[@Parkinson2020].

#### Tissue expression and enrichment

Transcript and protein expression data for prioritised genes were retrieved from the Human Protein Atlas (HPA, version 21.0)[@Uhlen2010]. We investigated transcript expression in a consensus RNA-seq dataset of 55 tissues, combining data from GTEx[@GTEx_Consortium2020; @Carithers2015] (RSEMv1.3.0 v8) and the HPA[@Uhlen2015], and in the HPA RNA-seq blood dataset[@Uhlen2019], containing expression levels in 18 immune cell types and total peripheral blood mononuclear cells. We calculated tissue enrichment scores, using the R package TissueEnrich[@Jain2019], for genes in comparison to all genes present in GTEx[@GTEx_Consortium2020; @Carithers2015] (RSEMv1.3.0 v8) and corrected for multiple comparisons using the Benjamini-Hochberg method[@Jain2019]. To investigate protein expression, we retrieved tissue-specific expression scores from the HPA[@Uhlen2015].

#### Functional enrichment

We performed functional enrichment of prioritised genes against the universe of all annotated genes using g:Profiler[@Raudvere2019]. The following data sources were used; Gene Ontology (GO) Biological Process[@Mi2019], Kyoto Encyclopaedia of Genes and Genomes (KEGG)[@Kanehisa2000], Reactome[@Gillespie2022], and WikiPathways[@Martens2021]. Multiple testing was corrected for using the g:SCS algorithm[@Raudvere2019], with a threshold of *P* \< 0.01. The input list was ordered by MAIC score. To address pathway redundancy, we used the output of the functional enrichment analysis as an input to Enrichment Map[@Merico2010]. We created networks based on a Jaccard similarity cutoff of 0.25 for each of; GO:Molecular Function, GO:Cellular Component, GO:Biological Process, Reactome, and Wiki Pathways. We then repeated this for all combined. Protein-protein interaction enrichment was performed using STRING v11[@Szklarczyk2019]. We included all possible interaction sources but specified a minimum interaction score of 0.9. We used the the whole annotated genome as the statistical background. Markov Clustering Analysis (MCL) was applied to the resulting network with an inflation parameter of 3. Clusters were annotated with the Reactome pathway with the smallest False Discovery Rate.

#### *In-silico* perturbation

#### Software and code availability

MAIC is implemented in Python v3.9.7 (Python Software Foundation, Wilmington, United States). All other analyses were performed with R v4.2.2 (R Core Team, R Foundation for Statistical Computing, Vienna, Austria). All code required to reproduce the analyses is available at <https://github.com/JonathanEMillar/ards_maic_analysis>.

\newpage

## References

::: {#refs}
:::



