---
title: "The genomic landscape of Acute Respiratory Distress Syndrome: a meta-analysis by information content of genome-wide studies of the host response."
format: 
  pdf:
    include-in-header: 
      text: |
        \usepackage{lscape}
        \newcommand{\blandscape}{\begin{landscape}}
        \newcommand{\elandscape}{\end{landscape}}
    keep-tex: true
mainfont: Times New Roman
fontsize: "11"
mainfontoptions: 
  - Numbers=Lowercase
  - Numbers=Proportional
linestretch: "1.5"
cap-location: bottom
tbl-cap-location: top
fig-align: center
documentclass: article
papersize: "A4"
geometry: 
  - top=15mm
  - left=22.5mm
  - right=22.5mm
  - bottom=15mm
echo: false
bibliography: bib/ref_manuscript.bib
csl: bib/nature-medicine.csl
biblio-title: "References"
---

Jonathan E Millar^1,2^, Sara Clohisey-Hendry^1,2^, Megan McMannus^1^, Marie Zechner^1^, Bo Wang^1^, Nick Parkinson^1^, Melissa Jungnickel^1^, Nureen Mohamad Zaki^1^, Erola Pairo-Castineira^1,2^, Konrad Rawlik^1,2^, Joshua Rogers^1^, Clark D Russell^2^, Lieuwe DJ Bos^3^, Nuala J Meyer^4^, Carolyn Calfee^5^, Daniel F McAuley^6^, Manu Shankar-Hari^2^ and J Kenneth Baillie^1,2^

1.  Roslin Institute, University of Edinburgh, Edinburgh, United Kingdom.
2.  Baillie-Gifford Pandemic Science Hub, Centre for Inflammation Research, University of Edinburgh, Edinburgh, United Kingdom.
3.  Intensive Care, Amsterdam UMC-location AMC, University of Amsterdam, Amsterdam, The Netherlands.
4.  Division of Pulmonary, Allergy, and Critical Care, University of Pennsylvania Perelman School of Medicine, Philadelphia, Pennsylvania, USA.
5.  Division of Pulmonary, Critical Care, Allergy & Sleep Medicine, Department of Medicine, University of California San Francisco, San Francisco, California, USA.
6.  Wellcome-Wolfson Institute for Experimental Medicine, Queen's University Belfast, Belfast, United Kingdom.

\newpage

## Abstract

Acute respiratory distress syndrome (ARDS) is a clinically defined syndrome of acute hypoxaemic respiratory failure secondary to non-cardiogenic pulmonary oedema. It arises from a diverse set of triggers and encompasses marked biological heterogeneity, which has complicated efforts to develop effective therapies. Using using an algorithm called meta-analysis by information content (MAIC), we performed a systematic review and computational integration of historical omics data implicating host response pathways in ARDS pathogenesis. The review identified 40 studies published over 20 years reporting genome-wide associations for 6,856 ARDS patients. These encompassed diverse study designs including transcriptomics, proteomics, and genome-wide association studies. By comparing gene lists across heterogeneous data sources, MAIC ranked over 7,000 genes by weighted cumulative evidence of association. This approach prioritized 1,306 genes most strongly implicated in ARDS pathogenesis. Functional enrichment of these genes revealed cholesterol metabolism, endothelial dysfunction, and innate immune activation - particularly neutrophil degranulation - as key processes. The analysis also highlighted 51 potential therapeutic targets such as *IL-6* and *MAP3K14*. To explore biological heterogeneity, MAIC was repeated on subsets of studies stratified by their focus on ARDS susceptibility or outcomes. This revealed most susceptibility genes were identified in blood while outcome-associated genes were enriched in airway samples, suggesting tissue-specific genomic perturbations. Overall, this study represents the first large-scale integration of historical omics data in ARDS, enhancing our understanding of the genomic landscape by synthesis of decades of disperse data. The gene rankings and interactive portal provided will help researchers refine hypotheses, select candidates for functional validation, and identify potential therapeutic targets or repurposing opportunities. Overall, our findings and the publically available computational framework represent an open, evolving platform for ARDS genomics.

\newpage

## Introduction

The acute respiratory distress syndrome (ARDS) is clinically defined as acute hypoxaemic respiratory failure due to non-cardiogenic pulmonary oedema[@ARDS_Definition_Task_Force]. It occurs following a variety of insults; pulmonary and extra-pulmonary. While this definition has been useful in identifying patients at risk of serious morbidity and death[@Bellani2016], it overlooks the underlying biology and masks heterogeneity[@Wilson2020]. Arguably, this has contributed to limited success in developing therapeutics[@Laffey2018]. In contrast, a biological definition of ARDS, based on mechanistically distinct sub-phenotypes, may provide the lever necessary for future drug discovery[@Bos2022].

Functional genomics technologies enable disease characterisation at unprecedented resolution. The emergence of coronavirus disease 2019 (COVID-19) has provided an opportunity to test their usefulness for drug discovery. A notable success has been the finding that baricitinib, a Janus kinase inhibitor, reduces mortality in patients hospitalised with COVID-19[@RECOVERYBari2022]. *A priori* support for baricitinib was greatly enhanced following the discovery of a causal link between elevated tyrosine kinase 2 (*TYK2*) expression and severe COVID-19 in genome-wide association studies (GWAS)[@Pairo-Castineira2021; @Kousathanas2022]. The availability of omics data for non-COVID ARDS is limited by comparison, although recent studies have used these techniques to examine signatures of non-COVID ARDS sub-phenotypes[@Bos2019; @Sarma2022].

An unresolved challenge is how large omics data can be effectively exploited[@Gomez-Cabrero2014]. Specifically, how can we combine data from heterogeneous sources to derive new insights or recalibrate our current understanding in the light of new data? We have proposed meta-analysis by information content (MAIC) as a data-driven, algorithmic, method for combining gene lists from diverse sources[@Li2020]. MAIC is agnostic to the quality or methodology of the sources and combines ranked or un-ranked gene sets by calculating weights for each list and gene, and iteratively updating them to converge on a ranked meta-list. We have successfully applied MAIC to host-genomics studies of Influenza A[@Li2020] and SARS-CoV-2[@Pairo-Castineira2021; @Parkinson2020], and shown that it out-performs existing algorithms when combining ranked and un-ranked lists obtained from heterogeneous sources[@Wang2022-jb].

In this work, we present a living meta-analysis by information content of ARDS host genomics studies. This serves as an open-source resource for gene prioritisation, functional genomics, and drug target discovery. An interactive interface can be accessed at <https://baillielab.net/maic/ards>, alongside a complementary [R package](https://github.com/baillielab/ARDSMAICr). 

\newpage

## Results

#### Systematic review

Our search yielded 8,937 unique citations (Fig. S1). We retrieved 74 articles for full-text evaluation and included 40 in our meta-analysis[@Batra2022; @Bhargava2014; @Bhargava2017; @Bime2018; @Bos2019; @Bowler2004; @Chang2008; @Chen2013; @Chen2016; @Christie2012; @Dolinay2012; @Dong2013; @Englert2019; @Frenzel2011; @GuillenGuio2020; @Howrylak2009; @Jiang2020; @Juss2016; @Kangelaris2015; @Kovach2015; @Liao2021; @Lu2017; @Martucci2020; @Meyer2011; @Meyer2013; @Mirchandani2022; @Morrell2018; @Morrell2019; @Nick2016; @Nguyen2013; @Ren2016; @Sarma2022; @Scheller2019; @Shortt2014; @Wang2008; @Tejera2012; @Xu2021; @Zhang2021; @Zhang2022; @Zhu2017]. These 40 studies produced 44 unique gene lists (22 transcriptomic, 13 proteomic, and 9 based on genome-wide association studies (GWAS); see @tbl-tab1). Three studies reported results from multiple methodologies[@Liao2021; @Mirchandani2022; @Sarma2022], and several used more than one tissue type[@Bowler2004; @Chen2016; @Kovach2015]. Excluding GWAS, 14 gene lists (40%) were derived from lung or airway samples, and 21 (60%) from blood. We could not retrieve one gene list[@Frenzel2011]. No whole-genome sequencing GWAS were found, and only 36% (n=8) of transcriptomic lists used next-generation sequencing techniques. The earliest included study was published in 2004[@Bowler2004], however, almost half (n=19, 47.5%) were published in the last 5 years.

Most studies aimed to identify genes or proteins associated with ARDS susceptibility (n=27, 67.5%). The remainder examined associations with survival (n=6, 15%), sub-phenotype (n=4, 10%), disease progression (n=2, 5%), or severity (n=1, 2.5%). In total, studies included 6,856 patients with ARDS. 

#### Meta-analysis by information content (MAIC)

First, we analysed all 43 available gene lists using MAIC. Lists were categorised by method (i.e., GWAS, transcriptomics, and proteomics) and technique (e.g., RNA-seq, mass spectrometry; see Table 1). In total, we ranked 7,085 unique genes (or SNPs), with a median of 27 genes per list (range 1-4,954). The top 100 ranked genes are summarized in @fig-fig1. Most genes were found in a single category (n=5,866, 82.8%); only 157 (2.2%) were identified in ≥ 3 categories, with the maximum number of categories supporting a gene being 5 (@fig-fig1). Similarly, few genes (n=362, 5.1%) were identified by more than one method, with only *AKR1B10*, *HINT1*, *HSPG2*, *S100A11*, and *SLC18A1* present in transcriptomic, proteomic, and GWAS-based lists. To prioritise genes for further investigation, we used the unit invariant knee method[@Christopoulos2016-jb] to identify the inflection point in the MAIC score curve. This prioritised 1,306 genes with scores above this point (@fig-fig1). These genes were more likely to be found in ≥ 2 lists or categories and by more than one method (@fig-fig1).

![**Meta-analysis by information content**. (a) Heatmap of top 100 ranked genes showing MAIC score, highest score per category, and number of supporting lists. (b) UpSet plot of MAIC genes showing total numbers for each category combination, MAIC score distribution, and supporting lists. (b) Gene prioritization using the Unit Invariant Knee method. Intersection of lines identifies elbow point of best-fit curve. 1,306 genes in upper left quadrant were prioritied. (c) Strip plots comparing number of lists and categories/methods per gene between prioritized and deprioritized sets. ](./img/Figure_1.png){#fig-fig1}

To assess the influence of individual lists, we calculated the information content (IC), reflecting the sum of gene scores across all lists (@fig-fig2), and the information contribution (ICtb), measuring the sum of gene scores contributing to a gene's overall MAIC score. To obtain relative values, we divided the IC/ICtb for each list by the total. This demonstrated that only 10 lists (from 9 studies) contributed >1% of the total information by either metric (Tab. S2). Notably, the RNA-seq list from Sarma et al.[@Sarma2022] accounts for >50% of the total IC and ICtb, a function of its length. To account for this, we normalised relative IC/ICtb by the number of genes per list. Along with the proportion of replicated genes in each list, this provides an alternative perspective, with several proteomic studies ranking highly (@fig-fig2).

![**Attributing information in MAIC**. (a) Shared information content (IC) between gene lists. Links indicate absolute IC (sum of common gene scores) between studies. (b) Proportion of replicated genes. Circle diameter is logarithm (base 2) of gene number per list. (c) IC normalized by number of genes. Overlapping circles denote equal normalized IC and contribution (ICtb - sum of common gene scores contributing to MAIC), indicating all gene scores contributed to MAIC. (d) Shared IC between categories, scaled so links show fraction of total IC. (e) Shared IC between methods, scaled. (f) Shared IC between tissue types, scaled.](./img/Figure_2.png){#fig-fig2}

#### Comparison with existing ARDS sources and COVID-19

To place our meta-analysis results in context, we evaluated the overlap between the genes prioritised by MAIC and those from two established resources: BioLitMine[@Hu2020-dl], using an ARDS MeSH search, and the ARDS Database of Genes[@Quintanilla2021] (Fig. S2a and Fig. S2c). BioLitMine identified 271 ARDS-associated genes, of which 142 (52.4%) were present in our analysis. Almost half of the overlapping genes (n = 63, 44.4%) ranked within our prioritised set (Tab. S3). Of the 239 genes catalogued in the ARDS Database of Genes, 177 (74.1%) were also found in our study. However, both sources contain gene associations lacking genome-wide support. 

\newpage

\blandscape

| **Year** | **Study**                      | **Focus**      | **Definition** | **N**^a^ | **Method**      | **Technique**  | **Tissue** | **Cell type** |
|----------|--------------------------------|----------------|----------------|----------|-----------------|----------------|------------|---------------|
| 2022     | Batra[@Batra2022]              | Survival       | Berlin         | 24       | Proteomics      | Other          | Blood      |               |
|          | Mirchandani[@Mirchandani2022]  | Susceptibility | Berlin         | 22       | Proteomics      | Mass Spec      | Blood      | Monocytes     |
|          |                                |                |                |          | Transcriptomics | Microarray     | Blood      | Monocytes     |
|          | Sarma[@Sarma2022]              | Sub-phenotype  | Berlin         | 41       | Proteomics      | Other          | TA         |               |
|          |                                |                |                |          | Transcriptomics | RNA-seq        | TA         |               |
|          |                                |                |                |          | Transcriptomics | scRNA-Seq      | TA         | Immune cells  |
|          | Zhang[@Zhang2022]              | Susceptibility | AECC           | 11       | Transcriptomics | RNA-Seq        | Blood      | Exosomes      |
| 2021     | Liao[@Liao2021]                | Survival       | Either         | 390      | GWAS            | Genotyping     | Blood      |               |
|          |                                |                |                |          | Transcriptomics | RNA-seq        | Blood      | PBMCs         |
|          | Martucci[@Martucci2020]        | Sub-phenotype  | None           | 11       | Transcriptomics | Microarray     | Blood      |               |
|          | Xu[@Xu2021]                    | Survival       | Berlin         | 105      | GWAS            | WES            | Blood      |               |
|          | Zhang[@Zhang2021]              | Susceptibility | Berlin         | 5        | Transcriptomics | RNA-seq        | Blood      |               |
| 2020     | Guillen-Guio[@GuillenGuio2020] | Susceptibility | Berlin         | 633      | GWAS            | Genotyping     | Blood      |               |
|          | Jiang[@Jiang2020]              | Susceptibility | Berlin         | 3        | Transcriptomics | scRNA-seq      | Blood      | PBMCs         |
| 2019     | Bos[@Bos2019]                  | Sub-phenotype  | Berlin         | 210      | Transcriptomics | Microarray     | Blood      |               |
|          | Englert[@Englert2019]          | Susceptibility | Either         | 11       | Transcriptomics | RNA-seq        | Blood      |               |
|          | Morrell[@Morrell2019]          | Survival       | AECC           | 36       | Transcriptomics | Microarray     | BALF       | AMs           |
|          | Scheller[@Scheller2019]        | Susceptibility | None           | 6        | Transcriptomics | RNA-seq        | BALF       | EVs           |
| 2018     | Bime[@Bime2018]                | Susceptibility | Either         | 232      | GWAS            | Genotyping     | Blood      |               |
|          | Morrell[@Morrell2018]          | Susceptibility | Berlin         | 35       | Transcriptomics | Microarray     | BALF       |               |
| 2017     | Bhargava[@Bhargava2017]        | Survival       | AECC           | 36       | Proteomics      | Mass Spec      | BALF       |               |
|          | Lu[@Lu2017]                    | Susceptibility | AECC           | 12       | Transcriptomics | Microarray     | Blood      |               |
|          | Zhu[@Zhu2017]                  | Susceptibility | Berlin         | 199      | Transcriptomics | Microarray     | Blood      |               |
| 2016     | Chen[@Chen2016]                | Severity       | AECC           | 7        | Proteomics      | Mass Spec      | BALF/Blood |               |
|          | Juss[@Juss2016]                | Susceptibility | Berlin         | 23       | Transcriptomics | Microarray     | Blood      | Neutrophils   |
|          | Nick[@Nick2016]                | Sub-phenotype  | AECC           | 121      | Transcriptomics | Microarray     | Blood      | Neutrophils   |
|          | Ren[@Ren2016]                  | Susceptibility | Berlin         | 14       | Proteomics      | Other          | Blood      |               |
| 2015     | Kangelaris[@Kangelaris2015]    | Susceptibility | Berlin         | 29       | Transcriptomics | Microarray     | Blood      |               |
|          | Kovach[@Kovach2015]            | Susceptibility | AECC           | 18       | Transcriptomics | Microarray     | BALF/Blood | AMs           |
| 2014     | Bhargava[@Bhargava2014]        | Progression    | AECC           | 22       | Proteomics      | Mass Spec      | BALF       |               |
|          | Shortt[@Shortt2014]            | Susceptibility | AECC           | 213      | GWAS            | WES            | Blood      |               |
| 2013     | Chen[@Chen2013]                | Susceptibility | Berlin         | 11       | Proteomics      | Mass Spec      | Blood      |               |
|          | Dong[@Dong2013]                | Progression    | None           | 14       | Proteomics      | Mass Spec      | BALF       | AMs           |
|          | Meyer[@Meyer2013]              | Susceptibility | Berlin         | 661      | GWAS            | Genotyping     | Blood      |               |
|          | Nguyen[@Nguyen2013]            | Susceptibility | AECC           | 30       | Proteomics      | Mass Spec      | BALF       |               |
| 2012     | Christie[@Christie2012]        | Susceptibility | AECC           | 812      | GWAS            | Genotyping     | Blood      |               |
|          | Dolinay[@Dolinay2012]          | Susceptibility | AECC           | 35       | Transcriptomics | Microarray     | Blood      |               |
|          | Tejera[@Tejera2012]            | Susceptibility | AECC           | 1400     | GWAS            | Genotyping     | Blood      |               |
| 2011     | Frenzel[@Frenzel2011]          | Survival       | AECC           | 46       | Proteomics      | Mass Spec      | BALF       |               |
|          | Meyer[@Meyer2011]              | Susceptibility | AECC           | 1241     | GWAS            | Genotyping     | Blood      |               |
| 2009     | Howrylak[@Howrylak2009]        | Susceptibility | AECC           | 13       | Transcriptomics | Microarray     | Blood      |               |
| 2008     | Chang[@Chang2008]              | Susceptibility | None           | 20       | Proteomics      | Mass Spec      | BALF       |               |
|          | Wang[@Wang2008]                | Susceptibility | AECC           | 8        | Transcriptomics | Microarray     | Blood      |               |
| 2004     | Bowler[@Bowler2004]            | Susceptibility | AECC           | 16       | Proteomics      | Mass Spec      | BALF/Blood |               |
: Summary of studies and gene lists included in the systematic review {#tbl-tab1}

```{=tex}
\begin{scriptsize}
a - The number of patients with ARDS included in each study.
Abbreviations: AECC - American-European Consensus Conference; AMs - Alveolar macrophages; BALF - Bronchoalveolar lavage fluid; EVs - Extracellular vesicles; GWAS - Genome-wide association study; MS - Mass spectometry; PBMCs - Peripheral blood mononuclear cells; TA - Tracheal aspirate; WES - Whole-exome sequencing. 
\end{scriptsize}
```

\elandscape

\newpage

After correcting historical gene symbol aliases, we matched 4 additional genes from the BioLitMine search not initially found in the ARDS MAIC set. A further 104 genes from this search were supported by just a single publication (Fig S2b). For each of the remaining 21 genes, we obtained the 100 most co-expressed genes using ARCHS4[@Lachmann2018] (returning data for 18) and assessed the overlap of these sets with ARDS MAIC; two-thirds exhibited <50% overlap (Fig. S2b). Finally, we compared the overlap between genes ranked by ARDS MAIC and those identified in a previous MAIC of the host response to COVID-19[@Parkinson2020] (Fig. S2d). In total, 2,606 ARDS-associated genes (36.8%) were common to both analyses, of which 143 were prioritised by both (Fig. S2e).

#### Tissue and cell-specific expression

Although most gene lists were derived from blood sampling, most genes were identified in airway samples (n=5,847, 82.5%) (Fig. S3a). This was also true for the prioritized gene set, however most of these were also identified in blood (n=818, 62.6%) (Fig. S3b). For the genes uniquely identified in lists from blood samples (n=1,238),  almost three-quarters are known to be expressed in the lung (HPA scRNA-seq data, ≥ 5 normalised transcripts per million (nTPM)), with a quarter being highly-expressed (≥ 100 nTPM) (Fig. S3c). For prioritized lung genes, there is a wide variety of cell-specific expression (Fig. S3d). However, in the smaller set of prioritized genes identified only in blood, clusters of expression specific to neutrophils, T cells, and monocytes are evident (Fig. S3e). Cell-type specific gene enrichment analysis suggests innate immune as well as epithelial and endothelial cell types are enriched among genes identified in airway samples (Fig. S3f). However, enrichment of epithelial and endothelial cells is not evident for prioritized genes identified from blood sampling alone (Fig. S3g).    

#### Functional enrichment 

Having identified a set of prioritised genes, we undertook several functional enrichment analyses. First, we performed over-representation analysis (ORA). In Reactome, 51 terms were significantly enriched (*P* < 0.001) (@fig-fig3). As expected, neutrophil degranulation and several innate immune pathways (e.g., IL-10 signalling, interferon signalling, MHC II antigen presentation, TLR4 cascade) feature heavily. However, multiple pathways associated with cholesterol biology and metabolism (e.g., chylomicron assembly/remodelling, GLUT4 translocation, TP53 regulation of metabolic genes, insulin regulation) were also over-represented. Similarly, lipid and cholesterol metabolism, as well as hyperlipidaemia, were over-represented in KEGG and WikiPathways (Fig. S4a and Fig. S4b). In an enrichment analysis using the GWAS Catolog, the prioritised set of genes was associated with asthma (adult onset/time to onset), monocyte, lymphocyte, and eosinophil counts, aspartate aminotransferase levels, and levels of apolipoprotein A1 (Fig. S4d). 

Next, we used the prioritised set of genes to create a protein-protein interaction (PPI) network. We graph-clustered this network, identifying 48 clusters with ≥ 5 members. Among the 10 largest clusters, we found programs associated with the proteaosome, cholesterol metabolism, interferon signalling, IL-6 signalling, and the complement cascade (Fig. S5). We then sought to use the PPI network to identify hub genes using an ensemble of topological methods. This analysis suggested 51 genes central to the wider network (@fig-fig3). Clustering these genes alone identified 5 clusters, which may be associated with innate immune cytokine signalling, interferon signalling, MHC class II antigen presentation, PI3K-Akt signalling, and eukaryotic translation elongation (@fig-fig3). The majority of hub genes (n=31, 61%) are currently druggable and include targets such as *IL-6*, *IL-17A*, *IL-18*, and *MAP3K14*.      

![**Functional enrichment of prioritised genes**. (a) Significantly enriched Reactome terms (*P* < 0.01). Terms colored by parent class and size proportional to recall. (b) Euler diagram of the overlap of hub genes identified by five methods. MNC - Maximum Neighbourhood Component, MCC - Maximal Clique Centrality, DMNC - Density of MNC, EPC - Edge Percolated Component. (c) Protein-protein interaction (PPI) network of hub genes, clustered using the Markov Chain Algorithm. (d). Heatmap of common hub genes displaying tissue type(s), MAIC score, highest category score, supporting lists, and presence in the druggable genome.](./img/Figure_3.png){#fig-fig3}

#### Sub-groups

A source of tension in our approach is the balance between dispartiy in study designs (e.g., susceptibilty, sub-phenotype) and the requirement for sufficient data to make meaningful inferences. To address this, we undertook MAIC on subsets of gene lists, stratified by study focus. There were sufficient lists to make this tractable for studies focused on susceptibility to ARDS (n=28) and studies of ARDS survival and severity (n=7) (@fig-fig4). 

For susceptibility, there were 15 transcriptomic (54%), 7 GWAS (25%), and 6 proteomic lists (21%). MAIC ranked 2,096 genes (@fig-fig4). The majority of these (n=1,222, 58%) were unique to susceptibility-based lists (@fig-fig4). Most were identified in blood, with a small fraction found solely in airways samples. The inflection point method prioritised the top ranked 130 genes (Fig. S6a). In comparison to the BioLitMine search and the ARDS Database of Genes, 71/271 and 117/239 genes were found among the ARDS MAIC susceptibility set respectively (Fig. S6b). A microarray-based transcriptomic list from Juss *et. al.*[@Juss2016] accounted for more than half (54.7%) of the relative ICtb, with an additional 12 lists having a relative ICtb ≥ 1% (Tab. S4). ORA using Reactome, KEGG, and WikiPathways identified 25 significantly enriched pathways including multiple terms related to cholesterol metabolism and glycolysis (Fig. S7a). A consensus of topological models identified 7 hub genes within a PPI network of prioritised genes. These genes cluster in a single group, characterised as being related to cholesterol metabolism by several pathway databases (@fig-fig4).   

For survival, the 8 gene lists consisted of 3 transcriptomic lists (37.5%), 3 proteomic lists (37.5%), and 2 GWAS (25%). MAIC ranked 463 genes (@fig-fig4). Approximately half of these (n=238, 51%) were unique to survival-based lists. In contrast to the susceptibility analysis, most survival genes were found in airways samples. Thirty-three genes were prioritised (Fig. S6d). In total, 32/271 of the BioLitMine ARDS-associated genes and 23/239 of the ARDS Database of Genes genes were found among the ARDS MAIC survival set (Fig. S6e). The proteomic and transcriptomic lists from Bhargava *et.al*[@Bhargava2017] and Morrell *et. al*[@Morrell2019] each contributed approximately 30% of the relative ICtb (Tab. S5). IL-10 and IL-18 signalling pathways were both significantly enriched in ORA (Fig. S7c). Graph-based clustering of the prioritised set of survival genes identified a single large cluster of immune-related genes including, *IL-10*, *CXCL8*, *TNFRSF1A*, and *IL2RA* (Fig. S7d).

![**MAIC of sub-groups**. (a) Schematic of ARDS MAIC sub-group analyses. (b) Heatmap of top 50 ranked genes in the susceptibility set showing MAIC score, highest score per category, and number of supporting lists. (c) Heatmap of 16 ranked genes in the survival set with multi-list support showing MAIC score, highest score per category, and number of supporting lists. (d) Eular diagram of gene overlap between the susceptibility and survival sets and the remainder of genes. (e) Bar plots of the tissue type in which genes are identified. (f) plot comparing the ranks of susceptibility and survival prioritised genes with their ranks in the full iteration of ARDS MAIC. (g) Euler diagram of the overlap of hub genes identified by five methods. MNC - Maximum Neighbourhood Component, MCC - Maximal Clique Centrality, DMNC - Density of MNC, EPC - Edge Percolated Component and a protein-protein interaction (PPI) network of hub genes, clustered using the Markov Chain Algorithm - for susceptibility.](./img/Figure_4.png){#fig-fig4}

\newpage

## Discussion

Our systematic integration and re-analysis of more than 20 years of data is the first large-scale meta-analysis of the genomic landscape of ARDS. This implicates over 7,000 genes and prioritises 1,306. The  wide inclusion criteria capture a diverse range of study designs and methods; the subsequent application of MAIC establishes the sum of this knowledge, downgrading noisy, irrelevant, or low-quality information. These results have three main applications. First, they can be used to better understand the pathobiology of ARDS, providing a resource to prioritise future *in-vitro* and *in-vivo* studies and permitting comparisons between important sub-groups. Second, they prioritise therapeutic targets, serving as a source against which novel and repurposed treatments can be screened. Third, they serve as a base for quantifying the novelty or additive nature of future omics studies in ARDS.   

Our review included 40 studies with genome-wide hypotheses. These studies varied in their aims and methods, however, some key themes are of note. Most obviously, the rate at which this form of study is being published is increasing; half of all studies in the last 5 years and a quarter since 2020. Relatedly, there were few stduies which employed next-generation sequencing (NGS) techniques or equivalent, and only two single-cell RNA-seq studies. A partial explanation may be the emergence of COVID-19, which is likely to have consumed the attention of many research teams active in this field. It is not unreasonable to conclude that an increasing number of non-COVID ARDS single-cell and NGS studies will emerge in the coming years. This reinforces the requirement for methods capable of meta-analysing multi-omic data[@pmid35817619]. Less obviously, a minority of studies have sampled the lung in ARDS, with only four examining the bulk transcriptome in the distal airspace. Reliance on information derived from blood samples may present a skewed picture of the pathobiology of ARDS and represents a missed opportunity to identify novel targets in the lung[@pmid35274164]. 

The key advantage of the MAIC approach in this setting is its ability to integrate diverse data sources. Traditional methods of gene list meta-analysis rely on simple vote counting or robust rank aggregation[@pmid22247279]. Instead, MAIC applies a data-derived weighting to each gene list, allows the investigator to define granular categorisation 

Our prioritisation of genes simultaneously validates existing concepts of ARDS pathobiology and provides novel insights. The functional prominence of innate immunity and cytokine signalling - in particular neutrophil-related activity - is unsurprising[@pmid921049]. As is the high ranking of genes such as *CXCL8*[@pmid36725964], *IL-18*[@pmid37695136], *MMP9*[@pmid21565917], and *MUC1*[@pmid33536260]. However, other findings are intriguing. A single gene example is histidine triad nucleotide binding protein 1 (*HINT1*), ranked 10^th^ and one of only 5 genes to have support from GWAS, transcriptomics, and proteomics methods. To our knowledge, no plausible role for *HINT1* has previously been suggested in ARDS[@pmid16835243]. Another notable finding is the significant enrichment of cholesterol uptake, efflux, and esterification pathways among prioritised genes. especially in the sub-group analysis of susceptibility.  

Outcome genes showed greater overlap, indicating shared pathways determine mortality/severity once established. IL-10 and IL-18 enrichment aligns with their anti-inflammatory and epithelial repair roles. 

Our approach has limitations. We purposefully sought studies with genome-wide hypotheses, excluding single-gene or candidate genetics studies. In the case of a gene with extensive evidence from the latter, our methodology may underestimate its association with ARDS. However, these study designs are subject to other limitations, such as publication and investigator biases[@pmid34309672]. In our iteration of MAIC, we did not account for direction of expression or effect. For a given gene, if the direction of expression differs between studies, MAIC may overestimate the strength of evidence associated with that gene. The inability to account for directionality also limits the scope of functional enrichment analyses which can be performed. Similarly, the use of an unsupervised prioritisation threshold may influence the outcomes of downstream analyses, however principled the approach. Finally, the general paucity of data, and in particular the limited number of single-cell transcriptomics (or proteomics) studies, limits the depth of inference that can be made. It is likely that many pathological perturbations occur with cell-specificity, which may not be apparent in bulk analyses of heterogeneous tissues[@pmid34446707]. In future. the addition of data from these modalities may reveal precision targets.  

Crucially, we provide an open platform and associated tools to enable deeper mining of the output. This allows others to re-analyse the data based on alternative sub-group divisions or to integrate unseen information. Ongoing multi-omic data integration with this initial study will further enhance the resolution of the data and increase our confidence in the results.

In summary, by systematically integrating decades of ARDS genomics, this study improves the scope for gene prioritisation and enhances molecular pathophysiological clarity. Our results strongly implicate  interferon signalling and cholesterol metabolism dysregulation, providing a specific therapeutic targets. Enrichment patterns and sub-group differences also give clues to genomic drivers of susceptibility, outcomes, and mortality. This substantially improves our conceptualisation of the genomic landscape of ARDS, setting the stage for functional validation.  

\newpage

## Methods

The systematic review and meta-analysis protocol was registered with the International Prospective Register of Systematic Reviews (PROSPERO; CRD42022306270). The review is reported in compliance with the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) guidelines[@Page2021].

#### Search strategy and selection criteria

A detailed description of our search strategy and eligibility criteria is provided in the Supplementary Methods. Briefly, we searched MEDLINE, Embase, bioRxiv, medRxiv, the ARDS Database of Genes[@Quintanilla2021], and the NCBI Gene Expression Omnibus from inception to April 1^st^, 2023 without language restrictions. We also performed single-level backwards and forwards citation searches using SpiderCite[@Clark2020] and hand-searched recent review articles[@Battaglini2022; @Hernandez-Beeftink2019; @Reilly2017; @Zheng2022].

We included human genome-wide studies reporting associations between genes, transcripts, or proteins and ARDS susceptibility, severity, survival, or phenotype, accepting any contemporaneous ARDS definition. We excluded paediatric studies (age < 18 years), animal studies, *in-vitro* human ARDS models, candidate *in-vivo* or *in-vitro studies* (< 50 genes/proteins), candidate gene associations, and studies with < 5 patients per arm (except scRNA-seq).

#### Outcomes

We retrieved ranked lists of genes associated with the ARDS host response, preferring measures of significance and adjusted *P* values over raw *P* values when multiple ranking measures were used. We obtained both summary lists (all implicated genes) and author-defined subgroup lists. To combine subgroup lists into summary lists, we took the minimum *P* value or maximum effect size. We excluded genes below the author-defined threshold for significance/effect magnitude. If unavailable, we excluded genes with *P* > 0.05, z-score < 1.96, or log fold change < 1.5.

#### Study selection and data extraction

Article titles and abstracts from our search were stored in Zotero v6.0-beta (Corporation for Digital Scholarship, United States). Titles were initially screened by one author using Screenatron[@Clark2020]. Two authors then independently screened abstracts against eligibility criteria, with a third resolving inconsistencies. Full texts and supplements of eligible studies were retrieved and inclusion adjudicated by consensus.

Data were extracted by one author and cross-checked by a second. Gene, transcript, or protein identifiers were mapped to HGNC symbols or Ensembl/RefSeq equivalents if no HGNC symbol was available. Unannotated SNPs were searched in NCBI dbSNP. miRBase (University of Manchester, United Kingdom) provided miRNA symbols. For microarray probes without symbols, we used the DAVID Gene Accession Conversion tool (Laboratory of Human Retrovirology and Immunoinformatics, Frederick National Laboratory for Cancer Research, United States) to map them to HGNC symbols. We extracted information relating to study design, methodology, tissue/cell type, demographics, ARDS aetiology, risk factors, severity, and outcomes.

#### Meta-analysis by information content (MAIC)

The MAIC algorithm has been described in detail[@Li2020; @Parkinson2020; @Pairo-Castineira2021; @Wang2022-jb]. Full documentation and the source code are available at https://github.com/baillielab/maic. Briefly, MAIC combines ranked and unranked lists of related named entities, such as genes, from heterogeneous experimental categories, without prior regard to the quality of each source. The algorithm makes four key assumptions; (1) genes associated with ARDS exist as true positives, (2) a gene is more likely to be a true positive if it is found in more than one source, (3) the probability of being a true positive is enhanced if the gene appears in a list that contains a higher proportion of replicated genes, and (4) the probability is further enhanced if it is found in more than one category of experiment. Based on these assumptions, MAIC compares lists with each other, forming a weighting for each source based on its information content, which is then used to calculate a score for each gene. The output is a ranked list summarizing the total information supporting each gene's association with ARDS. We have shown MAIC outperforms available algorithms, especially with ranked and unranked heterogeneous data[@Wang2022-jb].

As our primary analysis, we performed MAIC on all summary gene lists, regardless of study focus. Lists were assigned categories based on their methodology and experimental technique: genome-wide association study (GWAS) - genotyping, GWAS - whole exome sequencing, transcriptomics - microarray, transcriptomics - RNA-sequencing (RNA-seq), transcriptomics - single cell RNA-seq (scRNA-seq), proteomics - mass spectometry, and proteomics - other. For secondary analyses, we performed MAIC on subsets of lists based on study focus (i.e., susceptibility to ARDS or survival/severity). 

In secondary analyses, we repeated this pipeline for gene lists arising from studies in which the focus was susceptibility to ARDS or ARDS survival/severity. 

For each MAIC iteration, we prioritised genes with sufficient evidentiary support for further study (i.e., the gene set before which information content diminished such that there was little/no corroboration for the remainder's ARDS association). We used the unit invariant knee method [@inflection; @Christopoulos2016-jb] to identify the elbow point in the best-fit curve of MAIC scores. Genes with values above this point were prioritized for downstream analyses.

#### ARDS literature and SARS-CoV-2 associations

We used BioLitMine[@Hu2020-dl] to query the NCBI Gene database for genes associated with the Medical Subject Heading (MeSH) term "Respiratory Distress Syndrome, Acute", generating a list of genes and publications. We descriptively compared the overlap between this list and the MAIC-ranked gene list. Similar comparisons were made between the ARDS MAIC results and the gene set in the ARDS Database of Genes[@Quintanilla2021] and a prior MAIC of SARS-CoV-2 host genomics[@Parkinson2020].

#### Tissue expression and enrichment

Transcript and protein expression data for genes included in ARDS MAIC were retrieved from the Human Protein Atlas (HPA, version 21.0)[@Uhlen2010]. We investigated mRNA expression in a consensus scRNA-seq dataset of 81 cells from 31 sources (<https://www.proteinatlas.org/about/assays+annotation#singlecell_rna>) and in the HPA RNA-seq blood dataset[@Uhlen2019], containing expression levels in 18 immune cell types and total peripheral blood mononuclear cells. To investigate protein expression, we retrieved tissue-specific expression scores from the HPA[@Uhlen2015]. We conducted cell-type specific enrichment analysis using WebCSEA[@WebCSEA] and extracted the top 20 general cell types for each query.  

#### Functional enrichment

We performed functional enrichment of genes against the universe of all annotated genes using g:Profiler[@Raudvere2019]. The following data sources were used; Kyoto Encyclopaedia of Genes and Genomes (KEGG)[@Kanehisa2000], Reactome[@Gillespie2022], WikiPathways[@Martens2021], and Gene Ontology[@GO]. Multiple testing was corrected for using the g:SCS algorithm[@Raudvere2019], with a threshold of *P* \< 0.01. Input lists were ordered by MAIC score were appropriate. In the case of GO cellular component terms, we used the REVIGO tool to perform multi-dimensional scaling of the matrix of all pairwise semantic similarities[@REVIGO]. Enrichment was also performed against the National Human Genome Research Institute GWAS Catalog[@GWAScatalog] using the Enrichr web-interface[@enrichr]. Protein-protein interaction enrichment was performed using STRING v11[@Szklarczyk2019]. We included all possible interaction sources but specified a minimum interaction score of 0.7. We used the the whole annotated genome as the statistical background. Markov Clustering Analysis (MCL) was applied to the resulting network with an inflation parameter of 3. Clusters were annotated by hand having considered enrichment against KEGG, Reactome, and WikiPathways. To identify hub genes within the PPI network, we used cytoHubba[@cytoHubba] and Cytoscape[@cytoscape]. The highest ranked genes by Maximum Neighbourhood Component (MNC), Maximal Clique Centrality (MCC), Density of MNC (DMNC), Edge Percolated Component (EPC), and node degree were retrieved. The intersecting genes of these methods were deemed hub genes. Hub genes were searched for in the Drug Gene Interaction Database[@DGidb] to identify if they were present in the druggable genome.  

#### Software and code availability

MAIC is implemented in Python v3.9.7 (Python Software Foundation, Wilmington, United States). All other analyses were performed with R v4.2.2 (R Core Team, R Foundation for Statistical Computing, Vienna, Austria). Code required to reproduce the analyses is available at <https://github.com/JonathanEMillar/ards_maic_analysis>. An R package (ARDSMAICR) containing the data used in this manuscript and several functions helpful in analyses is available at <https://github.com/baillielab/ARDSMAICr>. 

\newpage

## References

::: {#refs}
:::




